src/atc/sql/SqlExecutor.py

```diff
@@ -34,6 +34,10 @@ def execute_sql_file(self, file_pattern: str, exclude_pattern: str = None):
             exclude_pattern = exclude_pattern.replace("*", ".*")
 
         replacements = Configurator().get_all_details()
+        # replacements update with schema manager
+        # name: manager.get_all_spark_sql_schemas()
+        # manager asks table configurator for all keys, specifically if they have a
+        # schema
```

Comment:
What happens here...? Is this WIP?

Reply:
indeed this is WIP. We talked yesterday.